src/frontend/package.json

```diff
@@ -5,6 +5,7 @@
   "main": "front/index.tsx",
   "scripts": {
     "build": "webpack",
+    "copy-iframe-resizer": "cp node_modules/iframe-resizer/js/iframeResizer.min.js ../backend/marsha/static/js",
```

Comment:
you should also add a command `build-dev` thant run `build` and `copy-iframe-resizer`. `build-dev` is easier to remember